getting-started.rst

```diff
@@ -24,6 +24,66 @@ The aw-qt application is the easiest way to use ActivityWatch. It creates a tray
 
 Simply run the :code:`./aw-qt` binary in the installation directory (either from your terminal or on Windows by double-clicking). You now should see an icon appear in your system tray (unless you're running Gnome 3, where there is no system tray).
 
+
```

Comment:
This makes the "Usage" section incredibly long, move it to its own section in the bottom or something and simply refer and link to it here.
Most activitywatch users are not using Gnome 3 or even Linux so it does not make sense to extend this section to be this long.

Reply:
Add it inside a ".. note::" as the others below

Reply:
Yeah, it's more consistent this way. Is the formatting correct now? I've never used this kind of markup before and the GitHub "preview" is not that satisfying